astroid/brain/brain_scipy_signal.py

```diff
@@ -1,14 +1,9 @@
-# Copyright (c) 2019 Valentin Valls <valentin.valls@esrf.fr>
-# Copyright (c) 2020-2021 hippo91 <guillaume.peillex@gmail.com>
-# Copyright (c) 2020 Claudiu Popa <pcmanticore@gmail.com>
-# Copyright (c) 2021 Pierre Sassoulas <pierre.sassoulas@gmail.com>
-# Copyright (c) 2021 Marc Mueller <30130371+cdce8p@users.noreply.github.com>
-
 # Licensed under the LGPL: https://www.gnu.org/licenses/old-licenses/lgpl-2.1.en.html
 # For details: https://github.com/PyCQA/astroid/blob/main/LICENSE
-
+# Copyright (c) https://github.com/PyCQA/astroid/graphs/contributors
 
 """Astroid hooks for scipy.signal module."""

```

Comment:
@DanielNoord I can be removing the line break here. But it's hard to understand why it fail, I'm investigating. Maybe because of Windows line ending in this file. I thougth the line ending were normalized.

Reply:
(Ideally I'd like to have a line break between copyright and docstring, and I don't want to have a strange error reappear if someone add a line break later)

Reply:
Yeap, that was the reason, see https://github.com/PyCQA/astroid/pull/1443